run-tests.sh

```diff
@@ -3,9 +3,28 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# do nothing on Travis except for the master branch
+TRAVIS=${TRAVIS:-false}
+if [[ "$TRAVIS" == "true" && "$TRAVIS_BRANCH" != "master" ]]; then
+    echo "Skipping tests for branch $TRAVIS_BRANCH (not master)"
+    exit 0
+fi
+
 cd tests
+npm install node-fetch
```

Comment:
A `package.json` is the more conventional way to do this.

Reply:
Like https://github.com/whatwg/misc-server/pull/22?